src/main/java/org/janelia/saalfeldlab/paintera/viewer3d/OrthoSliceFX.java

```diff
@@ -200,12 +230,16 @@ private void initializeMeshes()
 		final PhongMaterial material = new PhongMaterial();
 		mv.setCullFace(CullFace.NONE);
 		mv.setMaterial(material);
-		material.setDiffuseColor(Color.BLACK);
-		material.setSpecularColor(Color.BLACK);
 
-		newMeshViews.add(mv);
+		// NOTE: the opacity property of the MeshView object does not have any effect.
+		// But the transparency can still be controlled by modifying the opacity value of the diffuse color.
+		material.setDiffuseColor(new Color(
+				textureDiffuseOpaqueColor.getRed(),
```

Comment:
Could we just use [`Color.deriveColor`](https://docs.oracle.com/javafx/2/api/javafx/scene/paint/Color.html#deriveColor(double,%20double,%20double,%20double)) here?
```java
textureDiffuseOpaqueColor.deriveColor(0.0, 1.0, 1.0, newv.doubleValue());
```